### PR TITLE
feat(oracle): Return names mapped by id

### PR DESCRIPTION
**Pull request by savudani8**

### CODE REVIEW

File: src/parachain/oracle.ts [hunk 1]
```diff
@@ -53,17 +46,13 @@ export interface OracleAPI {
      */
     getLastExchangeRateTime(): Promise<Date>;
     /**
-     * @returns An array with the oracle names
+     * @returns A map from the oracle's account id to its
```
Comment (theodorebugnet) [inline src/parachain/oracle.ts:49]: ...`name`

File: src/parachain/oracle.ts [hunk 2]
```diff
@@ -89,6 +78,11 @@ export interface OracleAPI {
      * @returns Convert a Satoshi amount to Planck
      */
     convertSatoshiToPlanck(satoshi: PolkaBTC): Promise<Big>;
+    /**
+     * @returns The period of time (in milliseconds) after an oracle's last submission
+     * during which it is considered online
+     */
+    getMaxDelay(): Promise<number>;
```
Comment (theodorebugnet) [inline src/parachain/oracle.ts:85]: I would name this something like `getOnlineTimeout`, what do you think?

**Review by theodorebugnet — APPROVED** [review]

File: src/parachain/oracle.ts (hunk 2 above)
Comment (nud3l) [reply src/parachain/oracle.ts:85]: it's in line with the name on the chain, so I think fine to keep

Comment (savudani8) [reply src/parachain/oracle.ts]: I agree with @theodorebugnet that it's more suggestive. We could change the naming in the parachain

Comment (nud3l) [reply src/parachain/oracle.ts]: ok, agree!

**Pull request merged**